src/StarterPlayer/StarterPlayerScripts/Client/Core/VFX/Characters/AokijiVFX.lua

```diff
@@ -60,8 +60,14 @@ end
 
 function GetMousePos(X,Y,Z,Boolean)
 	local RayMag1 = workspace.CurrentCamera:ScreenPointToRay(X, Y) 
-	local NewRay = Ray.new(RayMag1.Origin, RayMag1.Direction * ((Z and Z) or 200))
-	local Target,Position,Surface = workspace:FindPartOnRayWithIgnoreList(NewRay, {Character,workspace.World.Visuals})
+
+	local RayParam = RaycastParams.new()
+	RayParam.FilterType = Enum.RaycastFilterType.Exclude
+	RayParam.FilterDescendantsInstances = { Character, workspace.World.Visuals }
+
+	local RaycastResult = workspace:Raycast(RayMag1.Origin, RayMag1.Direction * ((Z and Z) or 200))
```

Comment:
Could you explain why `(Z and Z)` is there?

Reply:
Typo. Will fix.